internal/api/user.go

```diff
@@ -100,6 +100,12 @@ func (a *API) UserUpdate(w http.ResponseWriter, r *http.Request) error {
 		}
 	}
 
+	if user.HasMFAEnabled() && !session.IsAAL2() {
+		if (params.Password != nil && *params.Password != "") || params.Email != "" && user.GetEmail() != params.Email {
```

Comment:
I have no idea what goes first && or ||

Can you add some parentheses? Also the phone condition is missing?

Reply:
Do we want to classify phone as sensitive?